How To - Guides/How to run locally for testing/Running your own server.txt

```diff
@@ -0,0 +1,26 @@
+----------------------------------------------------------
+------------------- How to Run Locally -------------------
+----------------------------------------------------------
+
+First step is to download/clone the TG Station Repo, or your fork of it.
+Clone the repository to a folder named "tgstation".
+Open the folder you've cloned the repository to, and run the "BUILD.cmd" file. This will build the application for running - and will also update the server if you've made any changes.
```

Comment:
I don't think this is the case, we have hot reloading for ui (javascript) changes but not for anything else